Description: Move -Iinclude to CPPFLAGS
Author: Stefano Rivera <stefanor@debian.org>
Forwarded: http://news.gmane.org/find-root.php?message_id=%3c20110821220427.GC1738%40bach.rivera.co.za%3e
Last-Update: 2011-09-25

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -69,8 +69,8 @@
 JAVA_CLASSES = $(JAVA_SOURCES:.java=.class)
 JAVA_RUNTIME_CLASSES=$(JAVARUNTIME_SOURCES:.java=.class)
 
-CFLAGS=-Iinclude -O2
-CPPFLAGS=-W -Wall -Wmissing-prototypes -Wmissing-declarations
+CFLAGS=-O2
+CPPFLAGS=-W -Wall -Wmissing-prototypes -Wmissing-declarations -Iinclude
 
 all: snowball libstemmer.o stemwords $(C_OTHER_SOURCES) $(C_OTHER_HEADERS) $(C_OTHER_OBJECTS)
 
